src/main/resources/assets/api-docs.yaml

```diff
@@ -2049,6 +2049,51 @@ paths:
           description: The requested user doesnt exist in the system
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /api/nih-register/{userId}:
```

Comment:
nit: I can't remember this or find it in the comments, but I thought we discussed making this just `/api/nih` instead of `/api/nih-register`